src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -618,31 +618,31 @@
     <value>Crimson</value>
   </data>
   <data name="SteelBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Steel Blue</value>
+    <value>Steel blue</value>
   </data>
   <data name="MediumSeaGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Medium Sea Green</value>
+    <value>Medium sea green</value>
   </data>
   <data name="DarkOrangeColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dark Orange</value>
+    <value>Dark orange</value>
   </data>
   <data name="MediumVioletRedColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Medium Violet Red</value>
+    <value>Medium violet red</value>
   </data>
   <data name="DodgerBlueColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Dodger Blue</value>
+    <value>Dodger blue</value>
   </data>
   <data name="LimeGreenColorButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Lime Green</value>
+    <value>Lime green</value>

```

Comment:
I think all of these should stay as-is, actually. They're proper nouns of the color, so guidelines state we should capitalize all of it.